docs/proposals/storage_acls.md

```diff
@@ -0,0 +1,173 @@
+
+
+### Use Cases for ‘ACCESS’
+
+1. Use Case: Administrator wishes to creaxte user and permit the use of certain storage classes
+2. Use Case: Administrator wishes to create group and permit the use of storage classes
+3. Use Case: Cluster Admin wishes to allocate role “Storage Admin” to non-cluster Administrator for the purposes of creating storage
+
+## Use
+
+Use is the actual act of leveraging the storage feature once it’s enabled in your namespace. Use will be restricted by two methods. The SCC (Security Context Constraint) which as it is merged upstream will become the PSP (Pod Security Policy) and the use of ACLs controlled by Quota and Limits. 
+Kubernetes controls are managed differently than traditional storage in that we can restrict down to a file level if necessary. Though, it embraces the method of hierarchical permissions, it depends on administrators management of first and foremost, namespaces, to restrict users and functions. Therefore, the need to provide the same file system level security is eliminated by only granting users specific functions in that namespace.
+
+### Access to Physical Storage
+There are features of storage access we do wish to control at a more granular level. Today three characteristics of the file systems that controls user access:
+
+* FSGroups: This is the filesystem group permissions for block storage. The Storag or Cluster administrator is responsible for allocating the GID for the storage asset at creation
+* SELinux
+  User specific SELinux parameters are not currently supported with the ability to apply SELinux labels after provisioning storage
+* Supplemental Groups: This is the shared storage group permissions for the storage asset. These must match what is already created on the physical asset in order for the pod to access the storage
+
+**None of these parameters should be requested by the user...this should happen ABOVE the request for storage
+Range should not be so widely defined to enable users to access storage outside their circle
+
+In addition to applying GIDs some storage plugins require credentials for certain administrative functions.
+
+### Access to Storage Features
+Once the user is restricted based on the methods provided in “ACCESS” we want to further limit access to certain features. This has two distinct pieces. We want to limit:
+
+1. What a user is allowed to do:
+   * Quotas: provides constraints that limit aggregate resource consumption per project
+     * Persistent Volume Claims
+2. Ability to request specific storage 
+     * Storage Selectors
+3. Ability to provision storage dynamically:
+     * Storage Classes
+4. How much storage the user may consume: 
+      * Limits: enumerates resource constraints per project 
+       * Size of request
+       * Enforcement by Quota that calculates usage
+       * Number of requests/claims
+
+This PR will focus on providing a means  of Access Control Lists (ACLs) based on leveraging and extending Resource Quotas:
```

Comment:
this is contradictory to the PR i opened:

https://github.com/kubernetes/kubernetes/pull/34554

can you just link to that PR?